app/javascript/components/pages/Recipe.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import "./Recipe.css";
+
+export class Recipe extends Component {
+  render() {
+    return (
+      <div id="recipe-container">
+        <h1>Recipes</h1>
+        <h4>Banana Bread Snack Cakes</h4>
+        <img
+          src="https://cdn.pixabay.com/photo/2019/11/15/12/15/banana-4628285_960_720.jpg"
+          alt="Banana cake"
+        />
+        <h4>Ingredients</h4>
+        <p>1/3 cup mashed ripe bananas (about 1 small)</p>
```

Comment:
Question: Will recipe eventually come from the DB?

Reply:
No We just added it there.